src/callbacks_stage/positivity_adaptive_filter_dg1d.jl

```diff
@@ -0,0 +1,171 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
```

Comment:
I didn't have any test case... What is the usual way of testing/CI in Trixi.jl? I only checked shu-osher with and without filtering and it looks fine. I am planning to make a separate PR on some standard 1D euler test cases we used before, what do you think?

Reply:
We usually create an elixir and run it. We compute errors between the solution and the "initial condition", which we check against during the test. I can go over it with you on Slack in more detail.

Reply:
Thanks! Maybe we can also talk about testing in the meeting, I will not mark this as resolved for now.